google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/tag_templates_constants.py

```diff
@@ -0,0 +1,63 @@
+from google.cloud import datacatalog_v1beta1
+
+
+class TagTemplateConstants:
+    """
+    Constants from this class are used both by
+    prepare.DataCatalogTagTemplateFactory and by
+    respective tests. This class is intended to be the
+    source of truth regarding which fields are expected to exist
+    in the tag templates.
+    """
+
+    DISPLAY_NAME_IDX = 0
+    FIELD_TYPE_IDX = 1
+    IS_REQUIRED_IDX = 2
+
+    @staticmethod
+    def get_fields_dict_for_cluster_tag_templates():
```

Comment:
`get_fields_dict_for_cluster_tag_templates ` to  `get_fields_dict_for_cluster_tag_template` ?

Because you are creating just one tag template for the cluster right? This gives the impression that there will  be multiple.

same for methods below.